### DISCUSSION TITLE
Brainstorm 1

### DISCUSSION THREAD

**Discussion by federicoVisi** [federicoVisi/SoundS-gesture-sound-interaction#1] [Ideas]
A way to exchange ideas until the session on 14 July.

**Comment by janearnison** [comment]


**Reply by biancadagger** [reply]
heyy

**Reply by federicoVisi** [reply]
Yes I can

**Reply by MigueroCarudas** [reply]
namething

**Comment by federicoVisi** [comment]
![Screenshot 2022-06-30 at 18 20 10](https://user-images.githubusercontent.com/13439202/176728016-254b35e3-8dcb-433b-8384-ece52282cb8c.png)

**Comment by genesis-victoria** [comment]
hi ;)

**Comment by janearnison** [comment]
Hey all, 

I'm wondering if it might be possible to capture audio live and then somehow input it into something like federico's sampler in the machine learning patch. . Or do you have to drag and drop pre existing audio wav/aif files?

**Reply by janearnison** [reply]
Like using buffer~/record~ etc. . .

**Reply by federicoVisi** [reply]
Hi Jane,
Yes, you can definitely record live input to the `buffer~` inside the `GIMLeT.mangle` module using record~. Check the `record~` help file if you haven't already. The thing to pay attention to with the GIMLeT.mangle module is the name of the buffer being used. In order to make it possible to use more than one module at the same time, each instance of the module has a unique buffer name generated upon instantiation, you can see it yourself if you open the `bpathcer` (see picture). If you're interested in implementing a handy way of recording live input in the sampler buffer I can show you how to get the buffer name to record to automatically without having to dig into the module every time you want to do that.
<img width="1356" alt="Screenshot 2022-07-08 at 12 01 47" src="https://user-images.githubusercontent.com/13439202/177970762-da1d67aa-6673-4109-a023-72a90e01a80f.png">

**Reply by janearnison** [reply]
amazing thanks Federico. I'll open up the bpatcher and have a look around. Definitely interested for implementing this with automatic buffer naming !
